src/Amplitude.js

```diff
@@ -1,3 +1,10 @@
+/*
+ *  @module p5.sound
+ *  @submodule p5.sound
+ *  @for p5.sound
+ *  @main
```

Comment:
It looks like we just need a `@main` tag for the module in one spot: https://yui.github.io/yuidoc/syntax/index.html#maintag